docs/archive-data.md

```diff
@@ -0,0 +1,314 @@
+---
+id: archive-data
+title: How to archive data
+sidebar_label: Archive data
+---
+
+Data that is generated from a Workflow, such as Event Histories,  persists in normal Temporal storage per the configured retention period. The retention period exists so that the persistence store is not overwhelmed by the accummulation of Workflow data over time. To preserve Workflow data indefinitely, you can use the Archival feature which automatically moves the data from the normal persistence storage to long-term storage after the Workflow retention period expires.
+
+:::info
+
+**Archival is currently disabled by default.**
+- Archival is disabled when running Temporal via `docker-compose` and can not be enabled.
+- Archival is disabled by default when cloning the repo and installing the system manually, but can be enabled in the config.
+- Archival is disabled by default when deploying via [helm charts](https://github.com/temporalio/helm-charts/blob/d08ab3a582e9bc605da460b79cd7524e18d2c91d/templates/server-configmap.yaml#L176) but can be enabled in the config.
+
+:::
+
+:::info
+
+**Archival currently only supports Event Histories.**
+
+You may notice some boilerplate infrastructure to support archiving visibility records. This functionality is not yet supported and visibility records are currently deleted after the Workflow retention period.
+
+:::
+
+## Archiving use cases
+
+There are two main reasons you may want to keep data after the retention period has past:
+
+1. **Compliance**: For legal reasons, data may need to be stored for a long period of time.
+2. **Debugging**: Older data may help with debugging.
+
+## Archival components
+
+Archival consists of four main components.
+
+1. **Archival feature configuration**: Archival is controlled by the [`archival` configuration](https://github.com/temporalio/temporal/blob/master/config/development.yaml#L81) in the `config/development.yaml` file.
```

Comment:
```suggestion
1. **Archival configuration**: Archival is controlled by the [`archival` configuration](https://github.com/temporalio/temporal/blob/master/config/development.yaml#L81) in the `config/development.yaml` file.
```

Reply:
Also here you use the alternative version of github linking (master instead of specific commit) can we please make this consistent one way or the other

Reply:
> Also here you use the alternative version of github linking (master instead of specific commit) can we please make this consistent one way or the other

Linking to a commit was an oversight/mistake - we want to link to master.